### ISSUE TITLE
Catch errors when census api is down

### ISSUE THREAD

**Issue by mlslade** [fitnr/censusgeocode#3]
Tried sample program - get the following errors
```python
>>> from censusgeocode import CensusGeocode
>>> cg = CensusGeocode()
>>> cg.onelineaddress('1600 Pennsylvania Avenue, Washington, DC', returntype='locations')

Traceback (most recent call last):
  File "<pyshell#5>", line 1, in <module>
    cg.onelineaddress('1600 Pennsylvania Avenue, Washington, DC', returntype='locations')
  File "C:\Users\Mike\AppData\Local\Programs\Python\Python35-32\lib\site-packages\censusgeocode\censusgeocode.py", line 97, in onelineaddress
    return self._fetch('onelineaddress', fields, layers, returntype)
  File "C:\Users\Mike\AppData\Local\Programs\Python\Python35-32\lib\site-packages\censusgeocode\censusgeocode.py", line 66, in _fetch
    return CensusResult(response)
  File "C:\Users\Mike\AppData\Local\Programs\Python\Python35-32\lib\site-packages\censusgeocode\censusgeocode.py", line 106, in __init__
    self.input = data['result']['input']
KeyError: 'result'
```
Did I do something wrong?
thanks!

**Comment by fitnr** [comment]
The Census API doesn't have the best uptime - this looks like the result came back empty. I've been meaning to do better error catching when this happens. I'm changing the title of this issue to reflect the problem.

**Comment by mlslade** [comment]
Thanks for looking into this!

**Comment by fitnr** [comment]
This is fixed by 9957ab8. Update to version `0.3` to get the changes.
